.storybook/themes/commerce.scss

```diff
@@ -456,6 +456,40 @@
 				),
 			),
 			blocks: (
+				alert-bar: (
```

Comment:
We should add ```word-break: break-all;```

to fix the issue where words are causing the block to not scale correctly

<img width="408" alt="Google Chrome-2022-06-09-20-50 05" src="https://user-images.githubusercontent.com/868127/172932339-217c4f82-d3fb-4f82-87c0-5f9049f8bd79.png">

Reply:
`break-word` seems to work better (`break-all` even breaks the smallest of words and that's odd)